src/routes/venue/+page.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+	import InfoBox from '$lib/layout/InfoBox.svelte';
+	import PageLayout from '$lib/layout/PageLayout.svelte';
+	import location from './location.jpg';
+</script>
+
+<PageLayout>
+	<h1>Venue Information</h1>
+
+	<section>
+		<InfoBox>
+			<p>This year's venue will be in the upper two floors of the Kraftwerk.</p>
+			<p>
+				MaibornWolff GmbH<br />
+				Drygalski-Allee 25<br />
+				81477 München <br />
+				(<a href="https://maps.app.goo.gl/hUhbheFci2sSJ7SZ6">Google Maps</a>)
+			</p>
+			<p>
+				<img
+					src={location}
+					alt={'a comfortable-looking, modern seating area with windows in the background, showing a view over the city'}
+				/>
+			</p>
+			<p>
+				See some more impressions
+				<a
+					href="https://www.maibornwolff.de/location/muenchen-kraftwerk/"
+					rel="external noopener noreferrer"
+				>
+					here
+				</a>.
```

Comment:
This should remove some extra space in the link

```suggestion
				<a
					href="https://www.maibornwolff.de/location/muenchen-kraftwerk/"
					rel="external noopener noreferrer"
				>here</a>.
```